ingestion/src/main/java/feast/storage/service/ErrorsStoreService.java

```diff
@@ -38,7 +39,7 @@
 
   public static List<ErrorsStore> getAll() {
     return Lists.newArrayList(
-        Iterators.concat(manuallyRegistered.iterator(), serviceLoader.iterator()));
+            Iterators.concat(manuallyRegistered.iterator(), serviceLoader.iterator()));
```

Comment:
This is weird, are you using the google-java-format plugin or the imported googles styles xml?
https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml

The imports don't get rearranged for me either.